pkg/apis/hive/v1/dnszone_types.go

```diff
@@ -137,9 +134,9 @@ type GCPDNSZoneStatus struct {
 
 // AzureDNSZoneStatus contains status information to specific to Azure Cloud DNS Zones
 type AzureDNSZoneStatus struct {
-	// ZoneName is the name of the zone in Azure Cloud DNS
+	// ZoneID is the ID of the zone in Azure Cloud DNS
 	// +optional
-	ZoneName *string `json:"zoneName,omitempty"`
+	ZoneID *string `json:"zoneName,omitempty"`
```

Comment:
s/zoneName/zoneID/

Reply:
I'm sorry I didn't understand what you meant by this. I am going to remove this field entirely as it seems redundant so maybe what you're saying will already get resolved